envpool/sokoban/level_loader.cc

```diff
@@ -163,6 +165,7 @@ void LevelLoader::LoadFile(std::mt19937& gen) {
             << "x" << dim_room << std::endl;
         throw std::runtime_error(msg.str());
       }
+      levels_.emplace_back(std::make_pair(cur_level_idx++, cur_level));
```

Comment:
```suggestion
      levels_.emplace_back(std::make_pair(cur_level_idx++, std::move(cur_level)));
```

to avoid a copy of the vector. Though that may be optimized already by clang

https://en.cppreference.com/w/cpp/utility/move